config/settings.py

```diff
@@ -3,7 +3,7 @@
 
 class Settings(BaseSettings):
     database_url: str = "postgresql+asyncpg://authdb_owner:V97gEkaPchLf@ep-plain-water-a5scbycn.us-east-2.aws.neon.tech/authdb"
-    secret_key: str = "e705c2325058744e7328d842be8da79a1f7c934363b1f061c7934d4804f3d360"
+    secret_key: str = "empty_string_1"
```

Comment:
Storing sensitive information such as database credentials directly in the source code is a significant security risk. It exposes the database to potential unauthorized access if the codebase is compromised. Use environment variables to store sensitive information and ensure that the `.env` file is not included in version control.

**Label**: security

**Code suggestion**

```python
database_url: str = os.getenv("DATABASE_URL")
```